modules/porous_flow/include/materials/PorousFlowMaterial.h

```diff
@@ -81,5 +82,7 @@ class PorousFlowMaterial : public Material
   const VariableName _saturation_variable_name;
   const VariableName _temperature_variable_name;
   const VariableName _mass_fraction_variable_name;
-};
 
+  /// Unique Ids of supplied material properties
+  std::set<unsigned int> _prop_ids;
```

Comment:
why didn't you use `std::vector` ?   Just curious really - you could `getSuppliedItems` into a set, then `push_back` to a vector.

Reply:
I pretty much just copied the design for `_supplied_prop_ids` in Material.h/C.